prog/kubernetes/kubernetes_cluster_nexus.rb

```diff
@@ -13,6 +13,12 @@ def self.assemble(name:, kubernetes_version:, private_subnet_id:, project_id:, l
         fail "Invalid Kubernetes Version"
       end
 
+      subnet = Prog::Vnet::SubnetNexus.assemble(
```

Comment:
Should we name it as `#{CLUSTER_UBID/NAME}-subnet`

Reply:
motive? I guess to make it more legible for operations? I guess right now it's the default, right, unless I misread.

Reply:
> motive? 

Operational readability. When we have a private subnet in production, if it follows the `kc.....-subnet` pattern, we can easily identify it as a private subnet for a k8s cluster.

> I guess right now it's the default, right, unless I misread.

The default name of the private subnet is the first 8 characters of its UBID, so it's not very informative.